knowhere/index/vector_index/IndexDiskANN.h

```diff
@@ -101,6 +101,7 @@ class IndexDiskANN : public VecIndex {
     std::unique_ptr<diskann::PQFlashIndex<T>> pq_flash_index_;
 
     bool is_prepared_ = false;
+    int32_t num_threads_  = 0;
```

Comment:
initial value 0 is ok ?
should set to 1 ?

Reply:
This is not a default num, we will set it in `prepare()`, and without preparation, we will throw.